src/web/lib/ArticleRenderer.tsx

```diff
@@ -278,14 +278,14 @@ export const ArticleRenderer: React.FC<{
                     );
                 case 'model.dotcomrendering.pageElements.RichLinkBlockElement':
                     return (
-                        <div key={i} id={`rich-link-${i}`}>
+                        <aside key={i} id={`rich-link-${i}`}>
```

Comment:
Might it make sense to move this `aside` along with the key and id generation into the component itself?